src/main/java/org/llorllale/cactoos/matchers/Throws.java

```diff
@@ -48,13 +50,27 @@
  * @param <T> Type of the scalar's value
  * @since 1.0.0
  * @checkstyle ProtectedMethodInFinalClassCheck (200 lines)
+ * @checkstyle MemberNameCheck (200 lines)
+ * @checkstyle ParameterNameCheck (200 lines)
+ * @todo #108:30min Refactor 'Throws' to eliminate fields with compound names.
+ *  There are two fields with compound names in this class:
+ *  'messageExpectation' and 'expectationDescription'. Compound names indicate
+ *  that the scope is too large and so cohesion too low. Create a new nested
+ *  class, maybe 'MessageExpectation', and move these fields and related
+ *  logic to this class.
  */
+@SuppressWarnings({"PMD.AvoidDuplicateLiterals", "PMD.LongVariable"})
 public final class Throws<T> extends TypeSafeDiagnosingMatcher<Scalar<T>> {
 
     /**
-     * The expected exception message.
+     * The expectation which the exception message must match.
      */
-    private final String msg;
+    private final UncheckedFunc<String, Boolean> messageExpectation;
```

Comment:
@vzurauskas Although debatable, but those projects deprecate [compound words as variables](https://www.yegor256.com/2015/01/12/compound-name-is-code-smell.html), use just `message` or just `expectation`

Reply:
@ilyakharlamov Same reply.